src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
It is also necessary to take out random. And like random, we also need to create a ColorSupplier object, not extend it.